connector/manifest.xml

```diff
@@ -10,24 +10,39 @@
     <vendor name="vendor"/>
     <driver name="driver"/>
     <customizations>
-      <customization name="CAP_SELECT_INTO" value="yes"/>
-      <customization name="CAP_SELECT_TOP_INTO" value="yes"/>
+      <!-- Metadata -->
+      <customization name="CAP_FAST_METADATA" value="yes"/>
+      <customization name="CAP_QUERY_WHERE_FALSE_METADATA" value="yes"/>
+      <!-- Temporary Tables -->
       <customization name="CAP_CREATE_TEMP_TABLES" value="no"/>
+      <customization name="CAP_SELECT_INTO" value="no"/>
+      <customization name="CAP_SELECT_TOP_INTO" value="no"/>
+      <!-- String Splits -->
+      <customization name="CAP_SUPPORTS_SPLIT_FROM_LEFT" value="no"/>
+      <customization name="CAP_SUPPORTS_SPLIT_FROM_RIGHT" value="no"/>
+      <!-- Initial SQL-->
+      <customization name="CAP_SUPPORTS_INITIAL_SQL" value="yes"/>
+      <!-- Query -->
+      <customization name="CAP_QUERY_ALLOW_JOIN_REORDER" value="yes"/>
+      <customization name="CAP_QUERY_BOOL_IDENTIFIER_TO_LOGICAL" value="yes"/>
       <customization name="CAP_QUERY_BOOLEXPR_TO_INTEXPR" value="no"/>
+      <customization name="CAP_QUERY_CASE_OUT_NO_BOOL_OPS" value="yes"/>
+      <customization name="CAP_QUERY_GROUP_BY_ALIAS" value="yes"/>
       <customization name="CAP_QUERY_GROUP_BY_BOOL" value="yes"/>
-      <customization name="CAP_QUERY_GROUP_BY_DEGREE" value="yes"/>
+      <customization name="CAP_QUERY_GROUP_BY_DEGREE" value="no"/>
+      <customization name="CAP_QUERY_HAVING_UNSUPPORTED" value="no"/>
+      <customization name="CAP_QUERY_INCLUDE_GROUP_BY_COLUMNS_IN_SELECT" value="yes"/>
+      <customization name="CAP_QUERY_JOIN_PUSH_DOWN_CONDITION_EXPRESSIONS" value="yes"/>
       <customization name="CAP_QUERY_SORT_BY" value="yes"/>
+      <customization name="CAP_QUERY_SORT_BY_DEGREE" value="yes"/>
       <customization name="CAP_QUERY_SUBQUERIES" value="yes"/>
-      <customization name="CAP_QUERY_TOP_N" value="yes"/>
-      <customization name="CAP_QUERY_TOP_SAMPLE" value="yes"/>
-      <customization name="CAP_QUERY_TOP_SAMPLE_PERCENT" value="yes"/>
-      <customization name="CAP_QUERY_WHERE_FALSE_METADATA" value="yes"/>
       <customization name="CAP_QUERY_SUBQUERIES_WITH_TOP" value="yes"/>
-      <customization name="CAP_SUPPORTS_SPLIT_FROM_LEFT" value="yes"/>
-      <customization name="CAP_SUPPORTS_SPLIT_FROM_RIGHT" value="yes"/>
-      <customization name="CAP_SUPPORTS_UNION" value="yes"/>
-      <customization name="CAP_QUERY_ALLOW_PARTIAL_AGGREGATION" value="no"/>
-      <customization name="CAP_QUERY_TIME_REQUIRES_CAST" value="yes"/>
+      <customization name="CAP_QUERY_SUBQUERY_QUERY_CONTEXT" value="yes"/>
+      <customization name="CAP_QUERY_SUPPORT_EMPTY_GROUPBY" value="no"/>
+      <customization name="CAP_QUERY_TOP_0" value="no"/>
+      <!-- JDBC -->
+      <customization name="CAP_JDBC_QUERY_CANCEL" value="no"/>
```

Comment:
Perhaps we should file a ticket for this one? I think we _could_ support it in theory, even if we don't now

Reply:
Filed ticket to address this [SQL-752](https://jira.mongodb.org/browse/SQL-752)